### PR TITLE
Added arb

### PR DESCRIPTION
**Pull request by isuruf**
Depends on #953

### CODE REVIEW

**Comment by isuruf** [comment]
@fredrik-johansson, any interest in maintaining this recipe?

**Comment by isuruf** [comment]
I've removed make check since builds timeout. (Due to building and testing done 3 times.) I'll enable the checks once the feedstock is created.

**Comment by isuruf** [comment]
Ready for review

File: recipes/arb/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+chmod +x configure
+
+# This is set to reduce the number of random tests run so that CIs can run
+# tests to completion without timeouts.
+export ARB_TEST_MULTIPLIER=0.1;
+
+./configure --prefix=$PREFIX --with-gmp=$PREFIX --with-mpfr=$PREFIX --with-flint=$PREFIX
+make
+#make check
```
Comment (ocefpaf) [inline recipes/arb/build.sh]: Just add a comment why you chose to not run the tests here.

Comment (ocefpaf) [inline recipes/arb/build.sh]: No need. Just saw that in your other message :smile:

**Comment by isuruf** [comment]
Thanks

**Pull request merged**